packages/health-check/Dockerfile

```diff
@@ -2,14 +2,14 @@ FROM node:16.1.0-alpine
 
 WORKDIR /usr/app
 
+RUN echo '*/5 * * * * /usr/app/cli/run critical > /dev/stdout' >> /etc/crontabs/root
```

Comment:
Follow Up: Should add some comments to this dockerfile explaining the steps and why the order is important.